### PR TITLE
CEP-468: Fix Dynamo methods

### PR DESCRIPTION
**Pull request by dlazarenko-macrometa**
It is the same PR as https://github.com/Macrometacorp/c84j/pull/97 but for `master`

Description:
Fixed All methods for Dynamo collection. Changed signature of the methods as it was wrong.
As Dynamo API is too simple. It consists only one endpoint for everything. Also, it consumes `Dynamo JSON`. It is specific type of `JSON` that requires converting in requests and back into responses.

Methods with new signture:

- `C8DynamoCreateTableEntity createTable(C8DynamoCreateTableOptions options)`
- `C8DynamoDeleteTableEntity deleteTable()`
- `C8DynamoDescribeTableEntity describeTable()`
- `C8DynamoPutItemEntity putItem(Map<String, Object> value)`
- `C8DynamoPutItemEntity updateItem(Map<String, Object> value)`
- `C8DynamoGetItemEntity getItem(Map<String, Object> key)`
- `C8DynamoDeleteItemEntity deleteItem(Map<String, Object> key)`
- `C8DynamoGetItemsEntity getItems(C8DynamoGetItemsOptions options)`
- `C8DynamoBatchWriteItemEntity batchWriteItems(Collection<Map<String, Object>> values)`

### CODE REVIEW

File: src/main/java/com/c8db/entity/C8DynamoBatchWriteItemEntity.java [hunk 1]
```diff
@@ -0,0 +1,23 @@
+/*
+ *
+ *  Copyright (c) 2022 Macrometa Corp All rights reserved
+ *
+ */
```
Comment (StoyanTc) [inline src/main/java/com/c8db/entity/C8DynamoBatchWriteItemEntity.java:5]: Please correct the header to be the same everywhere.

File: src/main/java/com/c8db/entity/C8DynamoBatchWriteItemInternalEntity.java [hunk 2]
```diff
@@ -0,0 +1,41 @@
+/*
+ *
+ *  Copyright (c) 2022 Macrometa Corp All rights reserved
+ *
+ */
```
Comment (StoyanTc) [inline src/main/java/com/c8db/entity/C8DynamoBatchWriteItemInternalEntity.java:5]: Too wide header, please fix.

File: src/main/java/com/c8db/entity/C8DynamoGetItemEntity.java [hunk 3]
```diff
@@ -0,0 +1,21 @@
+/*
+ *
+ *  * Copyright (c) 2022 Macrometa Corp All rights reserved
+ *
```
Comment (StoyanTc) [inline src/main/java/com/c8db/entity/C8DynamoGetItemEntity.java:4]: Please correct.

File: src/main/java/com/c8db/entity/C8DynamoGetItemsEntity.java [hunk 4]
```diff
@@ -0,0 +1,25 @@
+/*
+ *
+ *  * Copyright (c) 2022 Macrometa Corp All rights reserved
+ *
+ */
```
Comment (StoyanTc) [inline src/main/java/com/c8db/entity/C8DynamoGetItemsEntity.java:5]: Please correct.

File: src/main/java/com/c8db/entity/C8DynamoItemInternalEntity.java [hunk 5]
```diff
@@ -0,0 +1,35 @@
+/*
+ *
+ *  * Copyright (c) 2022 Macrometa Corp All rights reserved
+ *
+ */
```
Comment (StoyanTc) [inline src/main/java/com/c8db/entity/C8DynamoItemInternalEntity.java:5]: Please correct.

**Review by StoyanTc — APPROVED** [review]
@dlazarenko-macrometa - Please correct the headers on the newly created files to be the same everywhere.

File: src/main/java/com/c8db/entity/C8DynamoBatchWriteItemEntity.java (hunk 1 above)
Comment (dlazarenko-macrometa) [reply src/main/java/com/c8db/entity/C8DynamoBatchWriteItemEntity.java]: Done for all affected classes

**Pull request merged**